src/@types/react-bootstrap/index.d.ts

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Placement, PopoverProps } from 'react-bootstrap';
+
+// This overloads some types defined incorrectly in react-bootstrap.
+// See https://github.com/react-bootstrap/react-bootstrap/issues/4717
```

Comment:
Is there an issue in our repo to remember to remove this definition file once it is fixed in the upstream lib?

Reply:
Good call. https://github.com/mozilla/addons-code-manager/issues/1169